Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:14-alpine as nodejs
+
+FROM nodejs as builder
+
+WORKDIR /excalidraw-json
+
+ARG S3_ENDPOINT
```

Comment:
Many thanks for this! I was wondering though, would it be better to have environment variables that match the existing ones? Would mean one less thing to document.